.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!src
+!views
```

Comment:
the !views can be removed, we don't have that folder anymore

Reply:
views removed